adapters/pubmatic/pubmatictest/exemplary/banner.json

```diff
@@ -33,6 +33,13 @@
         "device":{
             "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36"
         },
+        "ext": {
+          "prebid": {
+            "bidderparams": {
+              "acat": ["drg”,”dlu”,”ssr"]
```

Comment:
I don't think this is testing what you think its testing. The inner double quotes are actually a different character such that this (and the same in other files) is an array of 1 item. Is that intentional?

Reply:
Fixed. Not sure how the quotes were changed. I doubt it was json validator plugin.
Thanks for pointing it out :)